android/src/main/java/io/parity/signer/screens/scan/camera/CameraView.kt

```diff
@@ -57,19 +68,32 @@ internal fun CameraViewInternal(viewModel: CameraViewModel) {
 
 				val imageAnalysis = ImageAnalysis.Builder()
 					.setBackpressureStrategy(ImageAnalysis.STRATEGY_KEEP_ONLY_LATEST)
+					.setOutputImageFormat(ImageAnalysis.OUTPUT_IMAGE_FORMAT_YUV_420_888)
+					.setTargetResolution(Size(1920,1080))
 					.build()
 					.apply {
 						setAnalyzer(executor) { imageProxy ->
 							viewModel.processFrame(barcodeScanner, imageProxy)
 						}
 					}
 
+				Log.e("rational is ", "${configuration.screenWidthDp} ${configuration.screenHeightDp}")
```

Comment:
We probably shouldn't leave that log here

Reply:
fixed